python_modules/dagster/dagster/_core/definitions/decorators/schedule_decorator.py

```diff
@@ -132,6 +138,9 @@ def inner(fn: RawScheduleEvaluationFunction) -> ScheduleDefinition:
         elif tags:
             validated_tags = validate_tags(tags, allow_reserved_tags=False)
 
+        context_param_name = get_context_param_name(fn)
```

Comment:
i know it's more wordy but `context_param_name_if_present` is more accurate and makes the need for subsequent truthy checks make sense. This also applies to the sensor PR